2333/2/libs/rapper.rb

```diff
@@ -0,0 +1,36 @@
+require 'russian_obscenity'
+
+# class Rapper
+class Rapper
+  attr_reader :name, :battles
+
+  def initialize(name, battles)
+    @name = name
+    @battles = battles
+  end
+
+  def stats
```

Comment:
I suggest you remove this method, it breaks encapsulation and encourages people to use objects of `Rapper` class in a weird way. In order to get data of a rapper, people should call methods of the rapper.